packages/lib/models/Setting.ts

```diff
@@ -991,6 +991,14 @@ class Setting extends BaseModel {
 				storage: SettingStorage.File,
 			},
 			'folders.sortOrder.reverse': { value: false, type: SettingItemType.Bool, storage: SettingStorage.File, isGlobal: true, public: true, label: () => _('Reverse sort order'), appTypes: [AppType.Cli] },
+			'folders.showNoteCount': {
```

Comment:
We already have the setting `showNoteCounts` so please use that. Just enable it for cli too, under appTypes

Reply:
Done [fdf2e3b](https://github.com/laurent22/joplin/pull/6526/commits/fdf2e3bf11119e2aca948866b715c2edbe3522b5)